tools/cli/commands/changelog.js

```diff
@@ -99,6 +99,28 @@ export function changelogDefine( yargs ) {
 					async argv => {
 						await changelogValidate( argv );
 					}
+				)
+				.command(
+					'write [project]',
+					'Writes all of the added changelog files to the project README.md',
+					yargWrite => {
+						yargWrite
+							.positional( 'project', {
+								describe: 'Project in the form of type/name, e.g. plugins/jetpack',
+								type: 'string',
+							} )
+							.option( 'dev', {
+								describe: 'Output validation issues using GitHub Action command syntax.',
+								type: 'bool',
+							} )
+							.option( 'prod', {
+								describe: 'Output file paths in this directory relative to it.',
+								type: 'bool',
+							} );
```

Comment:
Neither of these are valid options to `changelogger write`.

Reply:
Oh nice catch, I intended to fix those.